maintainers/maintainer-list.nix

```diff
@@ -6497,6 +6497,12 @@
     githubId = 127353;
     name = "Geoffrey Huntley";
   };
+  gigahawk = {
```

Comment:
SPLIT

Reply:
As in split the maintainers edit into a new commit?

Reply:
yep